sktime/tests/_config.py

```diff
@@ -39,7 +39,7 @@
     "EditDist",
     "CNNClassifier",
     "FCNClassifier",
-    "InceptionTimeClassifer",
+    "InceptionTimeClassifier",
```

Comment:
this has an impact on testing, but it is fine (the estimator should be excluded but was not)